.github/workflows/tests-postgres.yml

```diff
@@ -0,0 +1,118 @@
+name: Unit Tests Postgres
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+
+jobs:
+    phpunit:
+        name: PHPUnit
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: "8.0"
+                      db-image: 'postgres:14'
+                      reflector: "pdo"
+                      mode: "recording"
+                      dsn: "pgsql"
```

Comment:
why did you choose the term dsn?

In my book a dsn is a string containing db-type, db-host, db-name etc.

do we really need a separate parameter, or maybe we should have a `reflector: pdo-mysql` and a `reflector: pdo-pgsql`?

Reply:
Another point in favor of `pdo-pgsql` is, that we might get a separate `pgsql` reflector based on `pgsql_*` or other apis.

please use `pdo-pgsql` without a new DSN parameter